src/main/java/io/github/amerebagatelle/fabricskyboxes/mixin/skybox/WorldRendererAccess.java

```diff
@@ -26,8 +26,12 @@
     VertexBuffer getDarkSkyBuffer();
 
     @Accessor
-    Identifier getSUN();
+    static Identifier getSUN() {
```

Comment:
Why is this throwing a assertion error instead of just not existing?

Reply:
it's supposed to be a static accessor. 
it throws an `AssertionError` because it should never run